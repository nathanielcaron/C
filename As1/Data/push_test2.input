u 10
u 20
x