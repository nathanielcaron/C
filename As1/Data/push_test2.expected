10
20
Successfully executed 2 instructions
|_ 10 _|
|_ 20 _|
2 elements
